e2e/test-environment-esm/EnvESM.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import NodeEnvironment from 'jest-environment-node';
+
+export default class Env extends NodeEnvironment {
+  constructor() {
+    super();
+    this.global = global;
```

Comment:
you can do something like `this.global.someVar = 42` then in the test assert that `globalThis.someVar` is 42